heron/common/src/java/com/twitter/heron/common/basics/FileUtils.java

```diff
@@ -123,4 +123,17 @@ public static boolean isOriginalPackagePex(String packageFilename) {
   public static String getBaseName(String file) {
     return new File(file).getName();
   }
+
+  public static String getPkgType(String topologyBinaryFile) {
```

Comment:
@objmagic - Cause `String pkgType` is used for Config's key. Just in my opion, this scene is applicable to the principle of Occam's razor -- `If not necessary, do not increase the entity`